nimare/utils.py

```diff
@@ -1028,6 +1039,161 @@ def unique_rows(ar, return_counts=False):
         return ar[unique_row_indices]
 
 
+def _local_max(data, affine, min_distance):
```

Comment:
I like to include the permalink of the code I copied:
https://github.com/nilearn/nilearn/blob/8b16d36bbf0f1b88b9ccaea2da1f2867024e16d5/nilearn/reporting/_get_clusters_table.py#L26-L116

or in your case it may be difficult since you have an open pull request to nilearn so the permalink from nilearn does not represent the fix you made for the labels map, so disregard this comment when appropriate